android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/DateTimeExtension.kt

```diff
@@ -95,6 +95,13 @@ fun Date?.makeItReadable(): String {
   }
 }
 
+fun Date?.makeItReadable(inputFormat : String) : String {
+  return if (this == null) "N/A"
+  else {
+    SimpleDateFormat(inputFormat, Locale.getDefault()).run { format(this@makeItReadable) }
+  }
+}
```

Comment:
This function is duplicated.

Reply:
Yes, currently WIP. reverting some changes. thanks